src/elements/ConstF.H

```diff
@@ -100,19 +100,60 @@ namespace impactx::elements
             amrex::ParticleReal const betgam2 = powi<2>(refpart.pt) - 1.0_prt;
 
             // trigo
-            auto const [sin_kxds, cos_kxds] = amrex::Math::sincos(m_kx * slice_ds);
-            m_cos_kxds = cos_kxds;
-            m_const_x = -m_kx * sin_kxds;
-            auto const [sin_kyds, cos_kyds] = amrex::Math::sincos(m_ky * slice_ds);
-            m_cos_kyds = cos_kyds;
-            m_const_y = -m_ky * sin_kyds;
+            // Note: The convention here is m_kx[1/m] = sign(k)*sqrt(abs(k))
+            // where k[1/m^2] is in the usual MADX convention
```

Comment:
This statement is not clear, since there is not direct MADX analog.  For solenoids, the focusing strength is k [1/m].